content/technical/open-source-as-a-service.md

```diff
@@ -0,0 +1,155 @@
+Title: Your Service is not Open Source
+Date: 2018-02-06 18:35
```

Comment:
Do you have your time travel permit from the Ministry of Truth? :grin:

Reply:
Heh. Fixed.